hooks/src/index.js

```diff
@@ -341,7 +338,7 @@ export function useErrorBoundary(cb) {
 function flushAfterPaintEffects() {
 	let component;
 	while ((component = afterPaintEffects.shift())) {
-		if (!component._parentDom) continue;
+		if (!component._parentDom || !component.__hooks) continue;
```

Comment:
Is it possible to add a test for this?

Reply:
I tried but we consume our queue before 😅

Reply:
Ohh right makes sense. Forgot about our synchronous test logic.